server/public/javascript/sanitize.js

```diff
@@ -0,0 +1,15 @@
+const sanitizeEmails = (customers) => {
```

Comment:
why did you decide to place this function here? Don't you think this function should be part of the customer model?